src/games/strategy/triplea/ai/proAI/ProAI.java

```diff
@@ -140,7 +148,7 @@ protected void move(final boolean nonCombat, final IMoveDelegate moveDel, final
     BattleCalculator.clearOOLCache();
     ProLogUI.notifyStartOfRound(data.getSequence().getRound(), player.getName());
     initializeData();
-    ProBattleUtils.setData(data);
+    calc.setData(data);
```

Comment:
Why set the calc data here? The calc is not used in this method. This is an odd side effect of the "move" call.